_data/openssl/X509_V_ERR_CERT_NOT_YET_VALID.yml

```diff
@@ -0,0 +1,27 @@
+error-code: X509_V_ERR_CERT_NOT_YET_VALID
+slug: x509-v-err-cert-not-yet-valid
+tags: [ openssl, time ]
```

Comment:
Just wondering - Is the `openssl` tag still needed when it's made clear that this is an openssl error by its path?

Reply:
True, I'll delete that.